### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by eltoncezar**

### CODE REVIEW

File: SpotifyAPI/Local/Models/Track.cs [hunk 1]
```diff
@@ -157,5 +157,10 @@ public byte[] GetAlbumArtAsByteArray(AlbumArtSize size)
                 return wc.DownloadData(url);
             }
         }
+
+        public string GetId()
+        {
+            return this.TrackResource.Uri.Substring(14);
```
Comment (JohnnyCrazy) [inline SpotifyAPI/Local/Models/Track.cs]: While I think this is a good idea, I would suggest a class called `SpotifyUri` inside `Models` which parses the Uri into it's 3 strings, `Base` (will be always "spotify"), `Type` (like "album" or "track") and `Id`. 

Then we can implement a method inside `TrackResource` called `SpotifyUri ParseUri()` which returns a new instance of `SpotifyUri`.

**Comment by JohnnyCrazy** [comment]
Nice catch @eltoncezar !

I commented one line, maybe you can take a look at it. (I can also take care of it if you're busy or something)

**Comment by eltoncezar** [comment]
Sure, I can do it!

File: SpotifyAPI/Local/Models/SpotifyUri.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+{
+    public class SpotifyUri
+    {
+        public string Base { get; internal set; }
+        public string Type { get; internal set; }
+        public string Id { get; internal set; }
+
+        public SpotifyUri(string Uri)
+        {
+            string[] props = Uri.Split(':');
```
Comment (JohnnyCrazy) [inline SpotifyAPI/Local/Models/SpotifyUri.cs]: One thing we should propably do: Error-Handling

If `props.length` is not 3, throw a `InvalidOperationException` or sth similar (is there a better exception out there?)

Comment (eltoncezar) [inline SpotifyAPI/Local/Models/SpotifyUri.cs]: Indeed. Maybe an ```ArgumentException```?
There are other Uri types or formats that Spotify can return and we should check here?

Comment (JohnnyCrazy) [inline SpotifyAPI/Local/Models/SpotifyUri.cs]: Yea, `ArgumentException ` sounds like the correct way to go.
I'm not aware of them, but still, I think there is just no URI at all sometimes

**Comment by JohnnyCrazy** [comment]
Thanks, looks good!

**Comment by JohnnyCrazy** [comment]
After some thoughts, I moved the parsing to a static `Parse` method in 8d274164c4331865af0c158cc116976b62afff6c

**Pull request merged**